### DISCUSSION TITLE
Multi gpu setup question.

### DISCUSSION THREAD

**Discussion by RedDragonGecko** [turboderp/exllamav2#363] [General]
My current setup is a consumer board with two 3090's.
I'm upgrading to an Epyc system and looking to expand the gpu's
Looking to save a buck and drop in a few P100 16gb.
I'm assuming this wouldn't work and I need to stick with the same generation but thought I'd ask.

**Comment by turboderp** [comment]
It shouldn't technically be a problem to use GPUs from different generations as long as they're both supported. I'm not sure about the FP16 performance and VRAM bandwidth of the P100, though. :shrug:
